shared/macros-oval.jinja

```diff
@@ -534,3 +534,27 @@
         <description>{{{ description }}}</description>
     </metadata>
 {{%- endmacro %}}
+
+
+{{% macro bash_sssd_ldap_config(parameter, value) -%}}
+SSSD_CONF="/etc/sssd/sssd.conf"
+LDAP_REGEX='[[:space:]]*\[domain\/[^]]*]([^(\n)]*(\n)+)+?[[:space:]]*{{{ parameter }}}'
+AD_REGEX='[[:space:]]*\[domain\/[^]]*]([^(\n)]*(\n)+)+?[[:space:]]*id_provider[[:space:]]*=[[:space:]]*((?i)ad)[[:space:]]*$'
+DOMAIN_REGEX="[[:space:]]*\[domain\/[^]]*]"
+
+# Check if id_provider is not set to ad (Active Directory) which makes start_tls not applicable, note the -v option to invert the grep.
+# Try to find [domain/..] and {{{ parameter }}} in sssd.conf, if it exists, set to '{{{ value }}}'
+# if {{{ parameter }}} isn't here, add it
+# if [domain/..] doesn't exist, add it here for default domain
+if grep -qvzosP $AD_REGEX $SSSD_CONF; then
+        if grep -qzosP $LDAP_REGEX $SSSD_CONF; then
+                sed -i "s#{{{ parameter }}}[^(\n)]*#{{{ parameter }}} = {{{ value }}}#" $SSSD_CONF
+        elif grep -qs $DOMAIN_REGEX $SSSD_CONF; then
+                sed -i "/$DOMAIN_REGEX/a {{{ parameter }}} = {{{ value }}}" $SSSD_CONF
+        else
+                mkdir -p /etc/sssd
```

Comment:
This has been touched in #5997 - if a basic directory that should be owned by a package is not present, then it is probably better to assume that the remediation is premature, i.e. not applicable.

Reply:
+1

Reply:
I have removed the part of remediation that creates the file if doesn't exist similarly as it is done in #5997